packages/web-components/src/option/README.md

```diff
@@ -0,0 +1,139 @@
+| `checked`           | public    | `boolean or undefined`                                                                                         |         | The checked state is used when the parent listbox is in multiple selection mode. To avoid accessibility conflicts, the checked state should not be present in single selection mode. |
+| `content`           | public    | `Node[]`                                                                                                       |         | The default slotted content.                                                                                                                                                         |
+| `defaultSelected`   | public    | `boolean`                                                                                                      | `false` | The defaultSelected state of the option.                                                                                                                                             |
+| `disabled`          | public    | `boolean`                                                                                                      | `false` | The disabled state of the option.                                                                                                                                                    |
+| `selectedAttribute` | public    | `boolean`                                                                                                      |         | The selected attribute value. This sets the initial selected value.                                                                                                                  |
+| `selected`          | public    | `boolean`                                                                                                      |         | The checked state of the control.                                                                                                                                                    |
+| `dirtyValue`        | public    | `boolean`                                                                                                      |         | Track whether the value has been changed from the initial value                                                                                                                      |
+| `initialValue`      | protected | `string`                                                                                                       |         | The initial value of the option. This value sets the `value` property only when the `value` property has not been explicitly set.                                                    |
```

Comment:
Some of these could get confusing. There are *properties* and then *attributes* for all of these implementations. Something such as _initialValue_ might look like it's aligned to react...but the actual attribute here is _value_. The reason that the property is different is due to there being an implementation of a public getter and setter. I think we need a way here to enumerate what is a property what is an attribute, etc. I only see properties enumerated here.